providers/keycloak/provider.yaml

```diff
@@ -67,3 +67,11 @@ config:
         version_added: 1.0.0
         example: ~
         default: "http://host.docker.internal:48080"
+      all_admins:
```

Comment:
I am not against it, but do we want this option? I dont think it is needed

Reply:
I initially understood it as is, but then I misinterpreted after checking the SAM and thought we also want all admins here. I am okay to delete it. Also not strong opinion to keep it.

> (look how is made the SAM middleware when all admins is on)